GDAXSharp/Services/Orders/OrdersService.cs

```diff
@@ -89,6 +89,31 @@ public class OrdersService : AbstractService
             return await PlaceOrderAsync(order);
         }
 
+	    public async Task<OrderResponse> PlaceStopLimitOrderAsync(
```

Comment:
spacing here looks weird. should be consistent with the others. something like this:

````
        public async Task<OrderResponse> PlaceStopLimitOrderAsync(
            OrderSide side,
            ProductType productId,
            decimal size,
            decimal stopPrice,
            decimal limitPrice,
            bool postOnly = true,
            Guid? clientOid = null)
        {
            var order = new Order
            {
                Side = side,
                ProductId = productId,
                OrderType = OrderType.Limit,
                Price = limitPrice,
                Size = size,
                StopType = side == OrderSide.Buy
                    ? StopType.Entry
                    : StopType.Loss,
                StopPrice = stopPrice,
                ClientOid = clientOid,
                PostOnly = postOnly
            };

            return await PlaceOrderAsync(order);
        }
````